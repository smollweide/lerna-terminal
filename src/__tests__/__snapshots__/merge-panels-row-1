%7B%22merged%22:%5B%22%5Cu001b%5B37m%E2%94%8C%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B47m%20%5Cu001b%5B49m%5Cu001b%5B34m%5Cu001b%5B47m%5Cu001b%5B1mtitle%5Cu001b%5B22m%5Cu001b%5B49m%5Cu001b%5B39m%5Cu001b%5B47m%20%5Cu001b%5B49m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%90%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%8C%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B47m%20%5Cu001b%5B49m%5Cu001b%5B34m%5Cu001b%5B47m%5Cu001b%5B1mtitle%5Cu001b%5B22m%5Cu001b%5B49m%5Cu001b%5B39m%5Cu001b%5B47m%20%5Cu001b%5B49m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%90%5Cu001b%5B39m%22,%22%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%22,%22%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%20npm%20run%20start%20%20%20%20%20%20%20%20%20%20%20%20%20%20%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%20npm%20run%20start%20%20%20%20%20%20%20%20%20%20%20%20%20%20%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%22,%22%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%20server%20started%20%20%20%20%20%20%20%20%20%20%20%20%20%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%20server%20started%20%20%20%20%20%20%20%20%20%20%20%20%20%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%22,%22%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%22,%22%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%22,%22%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%22,%22%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%22,%22%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%5Cu001b%5B37m%E2%94%82%5Cu001b%5B39m%22,%22%5Cu001b%5B37m%E2%94%94%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%98%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%94%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%80%5Cu001b%5B39m%5Cu001b%5B37m%E2%94%98%5Cu001b%5B39m%22%5D%7D