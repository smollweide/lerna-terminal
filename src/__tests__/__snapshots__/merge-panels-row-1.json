{"merged":["┌── title ───────────────────┐┌── title ───────────────────┐","│                            ││                            │","│ npm run start              ││ npm run start              │","│ server started             ││ server started             │","│                            ││                            │","│                            ││                            │","│                            ││                            │","│                            ││                            │","│                            ││                            │","└────────────────────────────┘└────────────────────────────┘"]}